README.md

```diff
@@ -37,7 +37,7 @@ Here is a quick example of plotting an AIA image:
 ```python
 >>> import sunpy.map
```

Comment:
you will also need to add a `import sunpy.data.sample` line here to make this example complete.